bioio_base/reader.py

```diff
@@ -1089,12 +1089,25 @@ def standard_metadata(self) -> StandardMetadata:
                     metadata.objective = "40x Magnification"
                     return metadata
         """
+        # Attempt to get OME metadata; ignore if not implemented
+        try:
+            ome = self.ome_metadata
+        except NotImplementedError:
+            ome = None
+
         # Retrieve the dimensions information from the reader.
         dims = self.dims
         image_size_t = getattr(dims, DimensionNames.Time, None)
+
+        # helpers
+        from .standard_metadata import binning as _binning
```

Comment:
Why are these imported here rather than at the top of the file?

Reply:
resolved in [2171dc8](https://github.com/bioio-devs/bioio-base/pull/38/commits/2171dc8bf24c9ca10719acecd507c0c2f2cb4850)